resources/server/messages/messages.db.ts

```diff
@@ -102,6 +103,11 @@ export class _MessagesDB {
 
     const result = <ResultSetHeader>results;
 
+    const updateConversation = `UPDATE npwd_messages_conversations SET updatedAt = current_timestamp() WHERE id = ?`;
+
+    // We don't await as this isn't important to the return data
+    DbInterface._rawExec(updateConversation, [dto.conversationId]);
```

Comment:
Maybe its better to call a side effect function and await there just to not have an unhandled rejection, then you log it somewhere and keep moving from here.